src/store/client.ts

```diff
@@ -0,0 +1,7 @@
+import { createPublicClient, http } from 'viem';
+import { mainnet } from 'viem/chains';
+
+export const publicClient: ReturnType<typeof createPublicClient> = createPublicClient({
```

Comment:
I think `client` should go in `src/network` or more a location where we handle Data Layer stuff. cc @robpolak

Reply:
LGTM